files/build/scripts/buildinfo_base.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+BUILDINFO_PATH=/usr/local/share/buildinfo
+
+LOG_PATH=$BUILDINFO_PATH/log
+VERSION_PATH=$BUILDINFO_PATH/versions
+PRE_VERSION_PATH=$BUILDINFO_PATH/pre-versions
+DIFF_VERSION_PATH=$BUILDINFO_PATH/diff-versions
+BUILD_VERSION_PATH=$BUILDINFO_PATH/build-versions
+POST_VERSION_PATH=$BUILDINFO_PATH/post-versions
+VERSION_DEB_PREFERENCE=$BUILDINFO_PATH/versions/01-versions-deb
+
+. $BUILDINFO_PATH/config/buildinfo.config
```

Comment:
>buildinfo.config [](start = 25, length = 16)

I cannot find this file in this PR, or in the design doc.

Reply:
It is created in scripts/generate_buildinfo_config.sh as the config settings for wget/curl/pip.